index.html

```diff
@@ -15,22 +15,36 @@
     <header>
       <!-- NAVBAR -->
 
-      <nav>
+      <nav class="desktop-nav">
         <ul class="desktop-ul">
           <li class="navbar-link">
-            <a href="#recent-works-section">Portfolio</a>
+            <a href="#portfolio">Portfolio</a>
           </li>
           <li class="navbar-link">
-            <a href="#about-myself-section">About</a>
+            <a href="#about">About</a>
           </li>
           <li class="navbar-link">
-            <a href="#form-section">Contact</a>
+            <a href="#contact">Contact</a>
           </li>
         </ul>
         <ul class="mobile-ul">
           <li class="navbar-link"><a href="#">Welcome</a></li>
           <li class="navbar-link">
-            <a href="#"><img src="icons/menu-icon.png" alt="Icon" /></a>
+            <a class="open-menu-btn" href="#"><img src="icons/menu-icon.png" alt="Icon" /></a>
+          </li>
+        </ul>
+      </nav>
+      <nav class="menu visible">
+        <a class="close-menu-btn" href="#"><img src="icons/close-menu-icon.svg" alt="Icon"></a>
+        <ul>
+          <li>
```

Comment:
- [ ] Kindly make sure your closed button (X) is visible on the mobile menu size 👍🏻 
- [ ] Please let's shift the menu links down a little to give enough space to the close button which must be on the top 👍🏻 



![18](https://user-images.githubusercontent.com/79058364/188523175-6c129486-36cc-4cb3-b538-89d66a757bca.png)